src/compiler/instance.ts

```diff
@@ -100,9 +101,11 @@ const readThrough = (
   if (!cachedir) {
     return (code: string, fileName: string, lineOffset?: number) => {
       const normalizedFileName = normalize(fileName)
+
       logger.debug({ normalizedFileName }, 'readThrough(): no cache')
-      const [value, sourceMap] = compileFn(code, normalizedFileName, lineOffset)
-      const output = updateOutput(value, fileName, sourceMap, getExtension)
+
+      const [value, sourceMap] = compileFn(code, normalizedFileName, lineOffset),
+        output = updateOutput(value, fileName, sourceMap, getExtension)
```

Comment:
Is there a particular reason to initialize `output` in this manner? In ts-jest we have generally initialized/declared vars on their own and should keep this consistent. What do you think?

Reply:
Ya you are right I will change it to make it consistent 👍

Reply:
done